src/containers/profile/src/caseparticipation/CaseDetailsModal.js

```diff
@@ -147,7 +162,16 @@ const CaseDetailsModal = ({
               staffMemberByStatusEKID={staffMemberByStatusEKID} />
         </ModalSection>
         <ModalSection>
-          <header><Typography color={NEUTRAL.N700} variant="h3">Participants</Typography></header>
+          <header>
+            <Typography color={NEUTRAL.N700} variant="h3">Participants</Typography>
```

Comment:
I don't think this color will be applied.

Reply:
ah ya you are right—i'm just going to take this color out.